_docs/koppelaal-1.2/write-connector.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Write a Koppeltaal Connector
+category: How-to
+order: 2
+---
+
+# How-to: Koppeltaal connector
+
+## About this guide
+This guide will not go into Koppeltaal terminology, but rather assumes this knowledge is available to the reader. Whether it is from memory or the [Technical Design Document]. This document is not technical, but rather a descriptive / explanatory guide to assist the implementer towards a usable connector. Mainly by answering question in advance, that we think developers will bump into.
+
+## Connector repository and source code
+Since the connector will be used by other parties and will also be handed over to the Koppeltaal Community at some point, it’s important that the source code is clear and readable for other developers that want to debug or add features in the (near) future.
+
+A git repository can be created at the [Koppeltaal organization on GitHub]. Sources of connectors in other programming languages can be viewed there as well.
+
+## Where to start
+- Find out which FHIR resources need to be communicated about. Application dependent.
+- Is there FHIR model library in my programming language?
+
+## Connector responsibilities
+A Koppeltaal connector is a library to be used by an application written in particular programming language. It communicates directly with the Koppeltaal Server REST interface, while exposing a simple API to the application, exchanging custom objects. Koppeltaal specific complexity should be handled by the connector - e.g., composing a message that conforms to Koppeltaal ontology.
```

Comment:
written in particular programming language ==> written in a particular programming language